src/commands.rs

```diff
@@ -530,27 +533,40 @@ struct RestoreArgs {
     /// Revision to restore into (destination)
     #[arg(long)]
     to: Option<RevisionArg>,
-    /// Interactively choose which parts to restore
-    #[arg(long, short)]
-    interactive: bool,
     /// Restore only these paths (instead of all paths)
-    #[arg(conflicts_with = "interactive", value_hint = clap::ValueHint::AnyPath)]
+    #[arg(value_hint = clap::ValueHint::AnyPath)]
     paths: Vec<String>,
 }
 
-/// Touch up the content changes in a revision
+/// Touch up the content changes in a revision with a diff editor
+///
+/// With the `-r` option, which is the default, starts a diff editor (`meld` by
+/// default) on the changes in the revision.
+///
+/// With the `--from` and/or `--to` options, starts a diff editor comparing the
+/// "from" revision to the "to" revision.
 ///
-/// Starts a diff editor (`meld` by default) on the changes in the revision.
 /// Edit the right side of the diff until it looks the way you want. Once you
-/// close the editor, the revision will be updated. Descendants will be rebased
-/// on top as usual, which may result in conflicts. See `jj squash -i` or `jj
-/// unsquash -i` if you instead want to move changes into or out of the parent
-/// revision.
+/// close the editor, the revision specified with `-r` or `--to` will be
+/// updated. Descendants will be rebased on top as usual, which may result in
+/// conflicts.
+///
+/// See `jj restore` if you want to move entire files from one revision to
+/// another. See `jj squash -i` or `jj unsquash -i` if you instead want to move
```

Comment:
Should these perhaps be removed as well and replaced with `jj diffedit --to @-` and `jj diffedit --from @-`? (May or may not be part of this PR)

Reply:
I guess maybe not. Those are instances of `jj move` , not  `jj diffedit`.